__tests__/api.test.js

```diff
@@ -26,6 +26,42 @@ describe("/api/topics", () => {
   });
 });
 
+describe("/api/articles/:article_id", () => {
+  test("GET:200 return 1 article", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const article = body.article;
+        expect(article.article_id).toEqual(1);
```

Comment:
this should be .toBe to make your tests more secure, have a think about these other assertions and where you might be able to use .toBe